src/features/user/Settings/ImpersonateUser/SupportPin.tsx

```diff
@@ -12,13 +13,15 @@ interface SupportPin {
 const SupportPin = () => {
   const { token, user, setUser } = useContext(UserPropsContext);
   const { t } = useTranslation('me');
-
+  const { tenantConfig } = useTenant();
   const handleNewPin = async () => {
     try {
       const response = await putAuthenticatedRequest<SupportPin>(
+        tenantConfig?.id,
         '/app/profile/supportPin',
         undefined,
-        token
+        token,
+        undefined
```

Comment:
Resolve the warning:
`Argument of type 'undefined' is not assignable to parameter of type '(value?: string | undefined) => void'.`